cli/configuration.schema.js

```diff
@@ -0,0 +1,59 @@
+export default {
+  $schema: 'http://json-schema.org/schema#',
+  title: 'Configuration',
+  type: 'object',
+  required: ['baseUrl', 'jwt', 'mongo', 'storage'],
+  properties: {
+    baseUrl: {
```

Comment:
Shouldn't `baseUrl` be of _uri_ format? Also, I think it's generally a good idea to provide maxLength for strings wherever possible, e.g.

```
"baseUrl": {
  "type": "string",
  "maxLength": 255,
  "format": "uri"
}
```

Reply:
I was reluctant to add maxLength to uri strings as i have seen some pretty weird ones in the past. And restricting those to a specific length doesn't really provide much security or invalidation. I don't see why we should be strict for uri formats.

Reply:
OK, let's leave maxLength out for the moment.

If this string isn't URI-formatted, what could it be? Can you provide some examples?
imho this can only be a URI, e.g. `https://id.acme.com`

Reply:
No arguments on this being of uri format. I was only arguing on maxLength. Will add it